src/core/rendering.js

```diff
@@ -812,7 +812,7 @@ p5.prototype.clearDepth = function(depth) {
  *   stroke('#d41159');
  *   line(75, 25, 25, 75);
  *
- *   describe('A faint #1a85ff line and a faint #d41159 line form an X on a gray background. The area where they overlap is #B518B2.');
+ *   describe('An Ocean Blue line and a Hot pink line form an X on a gray background. The area where they overlap is Magenta purple.');
```

Comment:
There is inconsistency in capitalization of color names (e.g. "Ocean Blue" vs. "Hot pink."